atqa-framework/codecept.conf-FIREFOX.js

```diff
@@ -0,0 +1,58 @@
+require('ts-node/register');
```

Comment:
Is this file used somehow?

Reply:
yes, this config for runs FF via selenium standalone server

this should solve the problem with init extension on selenoid.